app/views/kaminari/ra-twitter-bootstrap/_gap.html.erb

```diff
@@ -1,5 +1,5 @@
-<li class="disabled">
-  <a href="#">
+<li class="page-item disabled">
+  <a href="#" class="page-link">
```

Comment:
Pagination structure was also updated in Bootstrap 5.
https://getbootstrap.com/docs/5.1/components/pagination/